### PR TITLE
Install additional development tools in base docker image

### PR DESCRIPTION
**Pull request by bhearsum**
This addresses #330 for the non-GPU tasks.

### CODE REVIEW

File: taskcluster/docker/base/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,13 @@ RUN apt-get update -qq \
                           python3-yaml \
                           locales \
                           git \
+                          tmux \
+                          htop \
```
Comment (hneiva) [inline taskcluster/docker/base/Dockerfile:31]: interesting that it requires htop

**Review by hneiva — APPROVED** [review]
lgtm!

**Comment by hneiva** [comment]
Just noticed `train-student-ru-en` failing. Seems unrelated..?

**Comment by eu9ene** [comment]
> Just noticed `train-student-ru-en` failing. Seems unrelated..?

I fixed it here
https://github.com/mozilla/firefox-translations-training/pull/373

File: taskcluster/docker/base/Dockerfile (hunk 1 above)
Comment (bhearsum) [reply taskcluster/docker/base/Dockerfile:31]: It's not the training process itself that requires htop; that's just a useful tool when folks are using interactive tasks to debug.

**Comment by bhearsum** [comment]
The quantize is #249 or #281, and certainly unrelated to this work - I'm going to go ahead and merge.

**Pull request merged**